### DISCUSSION TITLE
Group Messsages

### DISCUSSION THREAD

**Discussion by asadaly111** [munafio/chatify#342] [Q&A]
I have installed this chatify package but it is not working because i have seen many controllers and files in vendor folder so half code define and row query used there. So i think you need to improve more efficient code becuse now laravel 11 is launched already so update your package time by time.Now Group messaging feature is not available please help me.

**Comment by hussain4real** [comment]
> you

You can send a PR with better code organization

**Comment by munafio** [comment]
duplicated #222
